core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -1729,6 +1729,7 @@ private void validateLiteral(int pos, CharSequence tok) throws SqlException {
         columnAliasStop.add("from");
         columnAliasStop.add(",");
         columnAliasStop.add("over");
+        columnAliasStop.add("union");
```

Comment:
there is also `except` and `intersect` operations that i suspect will have the same problem

Reply:
Yes, that's what I'd also expect. Let me update the code and add some tests.

Reply:
Done in 9dba7f04ae7b56b2d31fd70111ccce0d170e077e